### ISSUE TITLE
[TensorFlow] No Java runtime present, requesting install

### ISSUE THREAD

**Issue by stu1130** [bytedeco/javacpp#417]
# Environment
1. Mac 10.14.6 (Mojave)
2. GraalVM 20.1.0 Java 11
# Step to reproduce:
```
1. install GraalVM following the [instruction](https://www.graalvm.org/docs/getting-started/macos)
2. git clone https://github.com/aws-samples/djl-demo.git
3. cd djl-demo/graalvm
4. mvn clean package -Pnative -Ptensorflow
5. target/image-classification
Loading:     100% |████████████████████████████████████████|
No Java runtime present, requesting install.
```
<img width="423" alt="Screen Shot 2020-07-27 at 10 15 54 AM" src="https://user-images.githubusercontent.com/6792331/88572045-63683e00-cff3-11ea-96be-d0e45325886d.png">

DJL uses TensorFlow JavaCpp bindings.
When building GraalVM with native mode, it shouldn't look for java during the runtime
Does JavaCpp for TensorFlow override the path of which JVM is loading?

**Comment by saudet** [comment]
Does the same thing happen with this sample project?
https://github.com/bytedeco/sample-projects/tree/master/opencv-stitching-native

**Comment by stu1130** [comment]
@saudet yes I got the same error

**Comment by saudet** [comment]
Hum, that's weird. It's not doing anything that should trigger something like that... 
I wonder if that gets triggered when trying to link with JavaVM framework but not installed?
https://github.com/bytedeco/javacpp/blob/master/src/main/java/org/bytedeco/javacpp/tools/Builder.java#L187

**Comment by saudet** [comment]
Yeah, it looks like that means the JDK isn't installed on the system. I wonder how GraalVM's JDK should be installed as a framework on a Mac for this to work correctly...

**Comment by stu1130** [comment]
The weirdest part is that my colleague @frankfliu don't have such issue, but me and one of the engineer from RedHat have the same error #https://github.com/awslabs/djl/issues/133

**Comment by saudet** [comment]
Oh, I see what's going on. Linking with the JavaVM framework actually makes it go look for `libjvm.dylib`, but it's not there unless the JDK is installed, which we don't need for the Java code after compiling everything AOT. Let me see how we can fix this...

**Comment by saudet** [comment]
I've fixed this in commit a59010d3c3b5881a9ef9eb4ebb1373a198652751, but to have it applied to TensorFlow, we need to upgrade to JavaCPP 1.5.4-SNAPSHOT.
@karllessard Are you OK with adding dependencies on snapshots?

/cc @johanvos

**Comment by karllessard** [comment]
When are you planning to release it? As long as TF Java is only available as a snapshot, it's fine but a release would be certainly better (I'm a bit biased in my statement because right now I'm building at the moment on a platform having issues to download snapshots from OSS Sonatype, are JavaCPP snapshots deployed on this repo as well?)

**Comment by saudet** [comment]
Sonatype OSSRH as well, yes: http://bytedeco.org/builds/

What kind of issues are you having?

**Comment by karllessard** [comment]
For our readers, @saudet and I discussed in private with the problems I'm having with Sonatype but regarding this issue, I think we are good to try this JavaCPP snapshot in TensorFlow for Java.

**Comment by stu1130** [comment]
Thanks for the quick fix. @saudet 
Is the latest patch available on 1.5.4-SNAPSHOT for macos?

**Comment by saudet** [comment]
Yes, see https://github.com/tensorflow/java/pull/93

**Comment by stu1130** [comment]
Thanks! I verified the problem has been solved.

**Comment by saudet** [comment]
Fix released with JavaCPP 1.5.4. Thanks for reporting this issue!
